### PR TITLE
Filter rows with `IS IN` enum values expression

### PR DESCRIPTION
**Pull request by billy1624**
## PR Info

With https://github.com/SeaQL/sea-query/pull/476, we'll have a fluent API, e.g.

```rs
assert_eq!(
    model,
    Entity::find()
        .filter(Column::Tea.is_in([Tea::EverydayTea.as_enum()]))
        .one(db)
        .await?
        .unwrap()
);
assert_eq!(
    model,
    Entity::find()
        .filter(Column::Tea.is_not_null())
        .filter(Column::Tea.is_not_in([Tea::BreakfastTea.as_enum()]))
        .one(db)
        .await?
        .unwrap()
);
```

### CODE REVIEW

**Review by ikrivosheev — APPROVED** [review]
@billy1624 wow! Really cool! Thank you!

**Comment by tyt2y3** [comment]
Worthy to add on cookbook

**Review by tyt2y3 — APPROVED** [review]

**Review by tyt2y3 — APPROVED** [review]

**Comment by tyt2y3** [comment]
Why 476 hasn't been merged but it still works?

File: tests/active_enum_tests.rs [hunk 1]
```diff
@@ -88,6 +93,29 @@ pub async fn insert_active_enum(db: &DatabaseConnection) -> Result<(), DbErr> {
...
+        Entity::find()
+            .filter(Column::Tea.is_not_null())
+            .filter(
+                Expr::col(Column::Tea)
+                    .binary(BinOper::NotIn, Expr::tuple([Tea::BreakfastTea.as_enum()]))
+            )
+            .one(db)
+            .await?
+            .unwrap()
+    );
```
Comment (billy1624) [inline tests/active_enum_tests.rs:118]: > Why 476 hasn't been merged but it still works?

This code snippet below is WITH https://github.com/SeaQL/sea-query/pull/476

```rs
assert_eq!(
    model,
    Entity::find()
        .filter(Column::Tea.is_in([Tea::EverydayTea.as_enum()]))
        .one(db)
        .await?
        .unwrap()
);
assert_eq!(
    model,
    Entity::find()
        .filter(Column::Tea.is_not_null())
        .filter(Column::Tea.is_not_in([Tea::BreakfastTea.as_enum()]))
        .one(db)
        .await?
        .unwrap()
);
```

**Comment by billy1624** [comment]
> Worthy to add on cookbook

Good call. I just added.
But we need this PR landed before publish it.
- https://github.com/SeaQL/sea-orm-cookbook/pull/1

**Comment by tyt2y3** [comment]
What I don't understand is, 476 has not been released yet, and I don't see any `git = ` in this PR. So I don't think it has a dependency on that?

**Comment by billy1624** [comment]
> What I don't understand is, 476 has not been released yet, and I don't see any `git = ` in this PR. So I don't think it has a dependency on it?

Correct! This PR does NOT depends on https://github.com/SeaQL/sea-query/pull/476

**Comment by tyt2y3** [comment]
I see the diff now. Yeah it's much cleaner!

**Comment by ikrivosheev** [comment]
@tyt2y3 hello! You released this, but SeaQuery doesn't... Something went wrong.

**Comment by ikrivosheev** [comment]
Ups, I did not see @billy1624 comment: https://github.com/SeaQL/sea-orm/pull/1183#issuecomment-1314715754.

Its ok)

**Comment by tyt2y3** [comment]
No worries

**Comment by billy1624** [comment]
Hahaa no problem!!

**Pull request merged**